__tests__/discord-groups/discord-group.test.js

```diff
@@ -0,0 +1,157 @@
+              'Access-Control-Allow-Origin': '*',
+              'Access-Control-Allow-Methods': 'GET, POST, PUT, DELETE, OPTIONS',
+              'Access-Control-Allow-Headers': 'Content-Type, Authorization',
+            },
+            body: JSON.stringify(allUsersData.users[0]),
+          });
+        } else if (url === `${BASE_URL}/discord-actions/groups`) {
+          interceptedRequest.respond({
+            status: 200,
+            contentType: 'application/json',
+            headers: {
+              'Access-Control-Allow-Origin': '*',
+              'Access-Control-Allow-Methods': 'GET, POST, PUT, DELETE, OPTIONS',
+              'Access-Control-Allow-Headers': 'Content-Type, Authorization',
+            },
+            body: JSON.stringify(discordGroups),
+          });
+        } else {
+          interceptedRequest.continue();
+        }
+      } else if (interceptedRequest.method() === 'POST') {
+        if (url === `${BASE_URL}/discord-actions/groups`) {
+          const postData = interceptedRequest.postData();
+          const groupData = JSON.parse(postData);
+          discordGroups.push(groupData);
+          interceptedRequest.respond({
+            status: 201,
+            contentType: 'application/json',
+            headers: {
+              'Access-Control-Allow-Origin': '*',
+              'Access-Control-Allow-Methods': 'GET, POST, PUT, DELETE, OPTIONS',
+              'Access-Control-Allow-Headers': 'Content-Type, Authorization',
+            },
+            body: JSON.stringify({ message: 'Group created successfully' }),
+          });
+        } else if (url === `${BASE_URL}/discord-actions/roles`) {
+          interceptedRequest.respond({
+            status: 201,
+            contentType: 'application/json',
+            headers: {
+              'Access-Control-Allow-Origin': '*',
+              'Access-Control-Allow-Methods': 'GET, POST, PUT, DELETE, OPTIONS',
+              'Access-Control-Allow-Headers': 'Content-Type, Authorization',
+            },
+            body: JSON.stringify({ message: 'Role created successfully' }),
+          });
+        } else {
+          interceptedRequest.continue();
+        }
+      } else {
+        interceptedRequest.continue();
+      }
+    });
+    await page.goto('http://localhost:8000/discord-groups');
+    await page.waitForNetworkIdle();
+    createGroup = await page.$('.create-groups-tab');
+    createGroupBtn = await page.$('#create-button');
+  });
+
+  afterAll(async () => {
+    await browser.close();
+  });
+
+  test('Page title should be "Discord Groups"', async () => {
+    const pageTitle = await page.title();
+    expect(pageTitle).toBe('Discord Groups | Real Dev Squad');
+  });
+
+  test('Create group button should be disabled for unverified users', async () => {
+    const isButtonDisabled = await page.$eval(
+      '.btn-create-group',
+      (button) => button.disabled,
+    );
+    expect(isButtonDisabled).toBe(false);
+  });
```

Comment:
the title says the button should be disabled but you are expecting it to be false?